README.md

```diff
@@ -62,6 +63,12 @@ report. Keep this in mind when setting the user roles which can access each repo
   instrument
   * Append *&as_image=1* to the report URL to retrieve as an image.
   * Access rights are still enforced, so reports may not be visible if embedded in surveys.
+* **Allow API access** \* determines whether the report can be accessed using the API
+  * Note that the Advanced Reports API is separate from the REDCap API and uses its own per-report
+    API keys.
+  * The API key has to be sent as the value of the `api_key` field in a HTTP POST request.
+  * Administrators have the option (in the module system settings) to prohibit non-administrators
+    from saving reports where API access is enabled.
 
 \* only available on some report types
```

Comment:
Please list the report types this is currently available on.

Reply:
I think providing a full list here might make this section of the documentation a bit unwieldly. Downloads, retrieval as image and API access are part of the common report options as they could apply to several report types, but they don't apply to all as sometimes they are not relevant (e.g. for PDF, accessing the report implies download, so an allow downloads option doesn't make sense).